jekyll/_cci2/webhooks.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: classic-docs
+title: "Webhooks"
+short-title: "Using Webhooks to subscribe to CircleCI events"
+description: "Using Webhooks to subscribe to CircleCI events"
+version:
+- Cloud
+---
+
+## Overview
+{: #overview}
+
+A webhook, sometimes referred to as a _reverse API_, allows you to connect a
+platform you manage (either an API you create yourself, or a third party
+service) to a stream of _future events_ (or, _real-time information_).
```

Comment:
Remove (or, _real-time information_).

Reply:
Should "future events" be italicized or only "events"? Only the latter seems to me like a distinct concept worth emphasizing.